components/AlertNotification/AlertNotification.tsx

```diff
@@ -46,6 +47,13 @@ const Alert = ({
       <Icon className="h-4 w-4" data-testid="alert-icon" />
       <AlertTitle title={title} data-testid="alert-title" />
       <AlertDescription message={message} data-testid="alert-message" />
+      <Button
+        variant="outline"
+        className="absolute right-4 top-2 p-1 h-8 bg-transparent"
```

Comment:
I feel like this is too far to the left. Can you make the spacing from the top and right side of the container equal?

Also, I believe you should be using the `icon` variant of the `Button` component.

Reply:
@kepsteen @ryandotfurrer - please remove Button from this ticket. It is out of scope, and I'm handling it on my ticket where it is explicitly declared. Thanks for the head start on the code though!

Reply:
@Danielle254 @ryandotfurrer Will do!